lib/lighthouse/benefits_claims/service.rb

```diff
@@ -25,31 +25,31 @@ def get_claims(lighthouse_client_id = nil, lighthouse_rsa_key_path = nil, option
       claims
     rescue Faraday::TimeoutError
       raise BenefitsClaims::ServiceException.new({ status: 504 }), 'Lighthouse Error'
-    rescue Faraday::ClientError => e
+    rescue Faraday::ClientError, Faraday::ServerError => e
```

Comment:
in Faraday 1+, 4XX-6XX errors are divided into these two categories, vs the `ClientError` that used to capture 4XX-6XX in <1